src/Service.Tests/Unittests/SqlQueryExecutorUnitTests.cs

```diff
@@ -318,6 +318,62 @@ Mock<MsSqlQueryExecutor> queryExecutor
             Assert.AreEqual(3, queryExecutorLogger.Invocations.Count);
         }
 
+        /// <summary>
+        /// Validates that a query successfully executes within two retries by checking that the SqlQueryExecutor logger
+        /// was invoked the expected number of times.
+        /// </summary>
```

Comment:
this description doesn't seem applicable to your new code. are you testing a specific function (and related downstream calls) within QueryExecutor? are you testing behavior interacting with a single specific helper function?

Reply:
Looks like "system under test (SUT)" is `msSqlQueryExecutor.StreamData(dbDataReader.Object, availableSize, new());` and you're just ensuring the SUT raises an exception when you need to stream data but availableSize is less than the amount you need to stream.

Reply:
fixed description.